test/peopleApi.spec.js

```diff
@@ -0,0 +1,43 @@
+/*global describe, it, beforeEach */
+
+var AlfrescoApi = require('../main');
+var AuthResponseMock = require('../test/mockObjects/mockAlfrescoApi').Auth;
+var PeopleMock = require('../test/mockObjects/mockAlfrescoApi').PeopleApi;
+
+describe('PeopleApi', function () {
+
+    beforeEach(function (done) {
+        this.hostEcm = 'http://127.0.0.1:8080';
+
+        this.authResponseMock = new AuthResponseMock(this.hostEcm);
+        this.peopleMock = new PeopleMock();
+
+        this.authResponseMock.get201Response();
+        this.alfrescoJsApi = new AlfrescoApi({
+            hostEcm: this.hostEcm
+        });
+
+        this.alfrescoJsApi.login('admin', 'admin').then(() => {
```

Comment:
Do you need this block of code for sure?

Reply:
removed